test/ussd_nurse_rapidpro.test.js

```diff
@@ -210,16 +210,35 @@ describe('app', function() {
                 });
 
                 describe('user enters msisdn', function(){
+
+                  //for msisdn that has not opted out
+                  it('should go to state_faccode', function(){
+                      return tester
+                      .setup.user.addr('27820001003')
+                      .inputs(
+                          {session_event: 'new'}
+                          ,'1' //chooses to start nurse connect
+                          ,'1'
```

Comment:
Please add comment on what this input is (similar to other test cases)

Reply:
Thanks, will add those. And regarding the description, you're right, that description explains the branch, and has no description for the pull-request